Untitled.py

```diff
@@ -0,0 +1,53 @@
+
+
+
+def sheet_data(sheet):
+
+    out = list()
+
+    sheetIndex = 0
+    for X in range(1, s.nrows):
+        learner = s.row_values(X)
+        print(learner)
+        average=sum(learner)
+        out.append([learner[1], average])
+
+    return out
+
+def sum(learner):
+    average = 0
+    for score in range(2, len(learner)) :
+        average = average+learner[score] // 2
+        if debug>0:
+            print(learner[score], average)
+    return average
+
+import xlrd
+
+a = xlrd.open_workbook("БИОЛОГИ_ таблица успеваемости осень 2018.xlsx")
```

Comment:
Non-ASCII characters (Cyrillic letters)

Reply:
I meant absolute path to our file. But now I think it will cause errors in case of file replacement, so using the file name is ok